src/components/patch/Patch.tsx

```diff
@@ -33,7 +34,8 @@ export class Patch extends React.Component<Props, State> {
       datetime: datetime,
       project: this.props.Patch.Version.identifier,
       author: this.props.Patch.Version.author,
-      builds: this.props.Patch.Builds
+      builds: this.props.Patch.Builds,
+      reconfigureLink: "https://evergreen.mongodb.com/patch/" + this.props.Patch.Version.id
```

Comment:
Get the root of the URL from the ui_url part of the config for this

Reply:
Fixed